control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2380,6 +2380,17 @@ components:
           type: string
       required:
         - id
+    CloneContentSource:
+      description: Volume Content Source i.e the snapshot and its source volume.
+      type: object
```

Comment:
Shouldn't this be an enum?
Containing either Snapshot { uuid, source_uuid } or Volume { uuid } ?

Reply:
Done